units/Cargo.toml

```diff
@@ -14,8 +14,8 @@ exclude = ["tests", "contrib"]
 
 [features]
 default = ["std"]
-std = ["alloc"]
-alloc = []
+std = ["alloc", "internals/std"]
+alloc = ["internals/alloc"]
```

Comment:
Actually you need to run `--no-default-features`.

Reply:
Still tests cleanly with no default features. Did you see the error or are you guessing from the diff?

Reply:
What kind of sorcery is this?! I was judging from the diff because there was no `#[cfg]` to remove the string where `InputString` is now. Which means my review wasn't very good and I should look into it better.

Reply:
There is still a bit of work to do to remove that TODO because of all the functions that return `String`.